agents/plugins/mk_mysql

```diff
@@ -27,11 +27,21 @@ function do_query() {
 
   echo "<<<mysql_slave>>>"
   echo "$INSTANCE_HEADER"
+  # do not fail but output nothing if we do not have a slave or do not have the required permission
```

Comment:
This has been addressed [here](https://github.com/tribe29/checkmk/commit/8caa532ad6f9639083c912683524e84fd85f7b31), but I think you might know this, since that was one day before your last post.

Reply:
No, I did not mention that. I'm putting my workaround in that plugin every time we do an update. But I will test it next time without my workaround.